www/lobby/id.ts

```diff
@@ -0,0 +1,63 @@
+import { Router, Request, Response, NextFunction } from 'express';
```

Comment:
merge with main first and then ill review this one!

Reply:
ok I merge main into this branch